applications/tari_merge_mining_proxy/src/helpers.rs

```diff
@@ -83,3 +86,24 @@ pub fn add_coinbase(
         Err(MmProxyError::MissingDataError("Coinbase Invalid".to_string()))
     }
 }
+
+pub fn default_accept(json: &Value) -> Value {
+    let id = json["id"].as_i64().unwrap_or_else(|| -1);
+    let accept_response = format!(
+        "{} \"id\": {}, \"jsonrpc\": \"2.0\", \"result\": {} \"status\": \"OK\",\"untrusted\": false {}{}",
+        "{", id, "{", "}", "}",
+    );
+    json::from_str(&accept_response).unwrap_or_default()
+}
```

Comment:
Generally, you should never use a serialised format such as JSON as a hard coded string in code. `serde_json` provides the `json!` macro for this.

```suggestion
pub fn default_accept(request: &Value) -> Value {
    json::json! {
        "id": request["id"],
        "jsonrpc": "2.0",
        "result" : json::json!(()),
        "status": "OK",
        "untrusted": false,
    }
}
```